client/src/docs-ui/secondary-nav/secondary-nav.tsx

```diff
@@ -1,33 +1,27 @@
-import {Component, Host, h, Prop, Build} from "@stencil/core";
+import * as links from "../../constants/links";
```

Comment:
NIT/NON-BLOCKING: I feel it would make more sense to import each individual constant. What was the reason on this?

Reply:
It's less writing / more readable imo